althea_kernel_interface/src/manual_peers_route.rs

```diff
@@ -0,0 +1,75 @@
+use super::{KernelInterface, KernelInterfaceError};
+
+use std::net::IpAddr;
+
+use failure::Error;
+
+impl KernelInterface {
+    pub fn manual_peers_route(
+        &self,
+        endpoint_ip: &IpAddr,
+        settings_default_route: &mut Vec<String>
+    ) -> Result<(), Error> {
+        // get default route (actually get all the routes)
+        let output = self.run_command(
+            "ip",
+            &[
+                "route",
+                "list",
+                "default",
+            ],
+        )?;
+
+        let stdout = String::from_utf8(output.stdout)?;
+
+        let endpoint_ip = format!("{}", endpoint_ip);
+
+        let mut def_route = Vec::new();
+
+        let mut found = false;
+
+        // find all lines
+        for i in stdout.split("\n") {
```

Comment:
I'd go with `lines()` here. I know this doesn't need to be too portable (only Linux really has iproute2), but I'd do it for the sake of sticking to good habits.

Reply:
good point